frontend/src/metabase/query_builder/components/expressions/ExpressionMode.js

```diff
@@ -0,0 +1,50 @@
+import "brace/mode/java";
+
+class ExpressionHighlight extends window.ace.acequire(
+  "ace/mode/text_highlight_rules",
+).TextHighlightRules {
+  constructor() {
+    super();
+
+    this.$rules = {
+      start: [
+        {
+          token: "string",
+          regex: '".*?"',
```

Comment:
I know single-quotes are converted onblur, but still
![image](https://user-images.githubusercontent.com/1447303/144909753-f26dea3d-67dd-4a65-a0e4-4123fdf510b3.png)

Reply:
Should be fixed now, thanks!